### PR TITLE
Issue32 add type annotations

### PR DESCRIPTION
**Pull request by tammymakesthings**
Fixes issue #32 
- Added type annotations to `adafruit_amg88xx.py`

### CODE REVIEW

File: adafruit_amg88xx.py [hunk 1]
```diff
@@ -23,56 +23,63 @@
...
-__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_AMG88xx"
+__version__: str = "0.0.0-auto.0"
+__repo__: str = "https://github.com/adafruit/Adafruit_CircuitPython_AMG88xx"
 
 from adafruit_bus_device.i2c_device import I2CDevice
 from adafruit_register import i2c_bit, i2c_bits
+from adafruit_register.i2c_bits import RWBits
+from adafruit_register.i2c_bit import RWBit
 from micropython import const
+import busio
```
Comment (tekktrik) [inline adafruit_amg88xx.py:34]: It looks like @FoamyGuy mentioned it in your other PRs, but this should go in the typing block!  I'm only reviewing it so it doesn't get merged in the meantime :)

**Review by tekktrik — CHANGES_REQUESTED** [review]

**Comment by tekktrik** [comment]
@FoamyGuy I think there's no harm in having the RWBit and RWBits outside of the try/except block here because they would have already been imported as part of `from adafruit_register import i2c_bit, i2c_bits` but not completely sure about that.  Thoughts?

File: adafruit_amg88xx.py (hunk 1 above)
Comment (tammymakesthings) [reply adafruit_amg88xx.py]: Requested change made. :-) Thanks for reviewing!

File: adafruit_amg88xx.py [hunk 2]
```diff
@@ -83,34 +96,34 @@ class AMG88XX:
...
+    _ihysh: RWBits = i2c_bits.RWBits(4, 0x0D, 0)
 
-    _tthl = i2c_bits.RWBits(8, 0x0E, 0)
+    _tthl: RWBits = i2c_bits.RWBits(8, 0x0E, 0)
 
-    _tthh = i2c_bits.RWBits(4, 0x0F, 0)
+    _tthh: RWBits = i2c_bits.RWBits(4, 0x0F, 0)
 
-    def __init__(self, i2c, addr=0x69):
+    def __init__(self, i2c: busio.I2C, addr: int = 0x69):
```
Comment (tekktrik) [inline adafruit_amg88xx.py:126]: Best practice is to add return type `None` for `__init__()`!

Comment (tammymakesthings) [reply adafruit_amg88xx.py]: Fixed that one, thanks!

**Pull request merged**